### PR TITLE
docs: Update references to Postgres 9.6

### PR DESCRIPTION
**Pull request by efritz**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ All notable changes to Sourcegraph are documented in this file.
 
 ### Changed
 
--
+- The minimum supported version of Postgres has been bumped from `9.6` to `12`. The upgrade procedure is mostly automated for existing deployments. Please refer to [this page](https://docs.sourcegraph.com/admin/postgres) for detailed instructions.
```
Comment (efritz) [inline CHANGELOG.md:22]: TODO: should update these instructions

File: doc/admin/postgres.md [hunk 2]
```diff
@@ -7,9 +7,9 @@ Sourcegraph uses several PostgreSQL databases to support various functionality.
 
 ## Version requirements
 
-> NOTE: ⚠️ From **3.27** onwards we will only support versions **starting from 12**.
+We support any version **starting from 12**.
 
-We support any version **starting from 9.6**.
+> NOTE: ⚠️ Version **3.26** required only Postgres 9.6. Check your database version before upgrading.
```
Comment (efritz) [inline doc/admin/postgres.md:12]: TODO: Forward reference to instructions

File: CHANGELOG.md (hunk 1 above)
Comment (davejrt) [reply CHANGELOG.md:22]: testing these now, will make any updates required

Comment (efritz) [reply CHANGELOG.md:22]: Feel free to commit directly to this branch if you want to consolidate doc updates.

**Review by davejrt — APPROVED** [review]

**Pull request merged**